readme.md

```diff
@@ -13,14 +13,12 @@ $ npm install --save taskkill
 ## Usage
 
 ```js
-var taskkill = require('taskkill');
+const taskkill = require('taskkill');
 
-var input = [4970, 4512];
+const input = [4970, 4512];
 
-taskkill(input, function (err) {
-	if (!err) {
-		console.log('Successfully terminated ' + input.join(', '));
-	}
+taskkill(input).then(() => {
+	console.log('Successfully terminated ' + input.join(', '));
```

Comment:
Might go for template strings here?

`console.log(`Successfully terminated ${input.join(', ')}`);`